main.py

```diff
@@ -9,7 +9,6 @@
 import kcidb_io
 import kcidb
 
-
```

Comment:
If this is needed for something like making pylint or some other linter happy, please move this change to the first commit, otherwise, please revert it.